sashimi-webapp/src/components/Content.vue

```diff
@@ -3,67 +3,43 @@
     <navbar></navbar>
     <div class="section group">
       <div class="col span_6_of_12">
-        <div>
-          <codemirror v-model="content" ref="myEditor" :options="editorOption" @changed="codeChange"> 
-          </codemirror>
-        </div>
+        <editor 
+          v-model="mdContent"
+          @input="updateContent"
+        ></editor>
       </div>
       <div class="col span_6_of_12">
-        <viewer></viewer>
+        <viewer :editor-content="mdContent"></viewer>
       </div>
     </div>
   </div>
 </template>
 
 <script>
-import { codemirror } from 'vue-codemirror';
-import wrapper from '../logic/wrapper';
 import navbar from './Navbar';
 import viewer from './Viewer';
 import editor from './Editor';
 
-let content = '';
-/* eslint : 0 */
-let codeMirrorInstance = null;
-
 /* eslint prefer-const: 0 */
 export default {
   components: {
-    codemirror,
     navbar,
     viewer,
     editor,
   },
-
   data() {
     return {
-      content,
-      editorOption: {
-        tabSize: 4,
-        mode: 'text/x-markdown',
-        theme: 'base16-dark',
-        lineNumbers: true,
-        lineWrapping: true,
-        line: true,
-        keyMap: 'sublime',
-        foldGutter: true,
-        gutters: ['CodeMirror-linenumbers', 'CodeMirror-foldgutter'],
-        styleSelectedText: true,
-        highlightSelectionMatches: { showToken: /\w/, annotateScrollbar: true },
-      }
+      mdContent: '',
     };
   },
-  methods: {
-    codeChange(newCode) {
+  method: {
+    updateContent(data) {
+      this.content = data;
```

Comment:
`updateContent` is not needed in this case. It appear to be the case where `v-model="mdContent"` is sufficient for the 2-way binding to work already.